### ISSUE TITLE
Explanation EDTF

### ISSUE THREAD

**Issue by maehr** [Stadt-Geschichte-Basel/forschung.stadtgeschichtebasel.ch#195]
### Describe what feature you'd like. Pseudo-code, mockups, or screenshots of similar solutions are encouraged!

We should add an explanation for EDTF

```edtf.md
Das [Extended Date Time Format (EDTF)](https://www.loc.gov/standards/datetime/) ist ein auf [ISO 8601 ](https://www.iso.org/iso-8601-date-and-time-format.html) aufbauender Standard, der über einfache Datumsangaben hinausgeht und es ermöglicht, komplexe, unsichere, ungefähre oder mehrdeutige Zeitangaben strukturiert und maschinenlesbar festzuhalten. Die wichtigsten Funktionalitäten und Merkmale sind:

1. **Einfache Datumsangaben**  
    Wie bei ISO 8601 üblich, kann man präzise Daten im Format `YYYY-MM-DD` angeben, z. B. `2024-12-11` für den 11. Dezember 2024.  
    Ebenfalls möglich sind Teilangaben wie `2024-12` (Dezember 2024) oder nur das Jahr `2024`.
    
2. **Ungewisse und ungefähre Daten**  
    Mit speziellen Zeichen lässt sich angeben, wenn ein Datum nur ungefähr oder unsicher ist:
    
    - `?` kennzeichnet Unsicherheit (z. B. `1960?` könnte bedeuten, dass es etwa 1960 war, aber nicht sicher ist).
    - `~` steht für ungefähre oder in etwa zutreffende Angaben (z. B. `1960~` bedeutet „ungefähr im Jahr 1960“).
    - `%` steht für unsicher und ungefähr
    
3. **Datumsbereiche und Intervalle**  
    Man kann Zeiträume als Intervalle ausdrücken, indem man zwei Daten mit einem Schrägstrich `/` verbindet:
    
    - `1900/1950` steht für die Zeitspanne von 1900 bis 1950.
    - Unvollständige Intervalle wie `1900/...` oder `.../1950` drücken offene Zeiträume aus („seit 1900“ oder „bis 1950“).
    
    Diese Intervalle können ebenfalls Unsicherheits- oder Ungefähr-Zeichen enthalten, um vage Zeiträume darzustellen: `1900~/1950?`.
    
4. **Listen und Sets von Daten**  
    Man kann mehrere mögliche oder relevante Daten in einer Liste angeben:
    
    - Listen: `[1960, 1962, 1965]` bedeutet eine explizite Aufzählung von möglichen oder relevanten Daten.

5. **Saisons und Quartale**  
    EDTF erlaubt die Angabe von Jahreszeiten oder Quartalen, die in manchen Zusammenhängen relevant sein können:
    
    - `1960-21` kann beispielsweise „Frühling 1960“ darstellen, wenn eine entsprechende Zuordnung der Jahreszeiten festgelegt wurde.
    - Der genaue Umgang mit Saisons ist festgelegt, um diese einheitlich anzugeben.
6. **Jahrhunderte, Jahrzehnte und unvollständige Angaben**  
    Statt eines konkreten Jahres kann man ganze Jahrzehnte oder Jahrhunderte angeben:
    
    - `19xx` für ein unbekanntes Jahr im 20. Jahrhundert.
    - `19xx?` für ein ungefähres oder unsicheres Jahr im 20. Jahrhundert.
    - `19uu` kann ähnlich verwendet werden, um Unsicherheit auf einer bestimmten Stellenebene auszudrücken.
7. **Angaben von Jahren kleiner als -9999 und grösser 9999**

   - `Y-30000` für 30'000 Jahre vor unserer Zeitrechnung.

**Zusammengefasst**: EDTF bietet weit mehr Möglichkeiten als ein einfaches Kalenderdatum. Es erlaubt die Erfassung von Unsicherheiten, Unschärfen, komplexen Zeiträumen und offenen Enden, sowie die Angabe von Mengen, Intervallen oder wiederkehrenden Mustern. Für Laien bedeutet das im Kern: Wo man bislang nur ein genaues Datum hatte, lassen sich jetzt auch „ungefähr“, „nicht genau bekannt“, „innerhalb eines Jahrzehnts“, „irgendwann zwischen X und Y“ oder „eine von mehreren Möglichkeiten“ in einem einheitlichen, standardisierten Format darstellen. Diese Flexibilität macht EDTF für historische, bibliothekarische, archivische oder wissenschaftliche Projekte besonders nützlich.
```

### What type of pull request would this be?

New Feature

**Comment by maehr** [comment]
```_data/config-metadata.csv
field,display_name,browse_link,external_link,dc_map,schema_map
objectid,Identifikator,,,DCTERMS.identifier,identifier
title,Titel,,,DCTERMS.title,headline
description,Beschreibung,,,DCTERMS.description,description
relation,Verwandte Ressourcen,,true,DCTERMS.relation,relation
subject,Thema,true,,DCTERMS.subject,keywords
era,Epoche,true,,DCTERMS.temporal,
isPartOf,Gehört zu,,true,DCTERMS.isPartOf,isPartOf
creator,Ersteller*in,,true,DCTERMS.creator,creator
date,Datum <a href='/edtf.html'>ℹ️<a>,,,DCTERMS.date,dateCreated
source,Quelle,,true,DCTERMS.source,
publisher,Verantwortliche Gedächtnisinstitution,,true,DCTERMS.publisher,publisher
type,Typ,,true,DCTERMS.type,
format,Format,,,DCTERMS.format,encodingFormat
extent,Auflösung,,,DCTERMS.exten🗓️t,
language,Sprache,true,,DCTERMS.language,inLanguage
rights,Rechte,,,DCTERMS.rights,usageInfo
license,Lizenz,,true,DCTERMS.license,license
```

**Comment by maehr** [comment]
@koilebeit 

```Metadatenmodel
Metadaten für die auf der Forschungsdatenplattform vorgestellten Elemente werden gemäß einem Datenmodell bereitgestellt, das vom Stadt.Geschichte.Basel Research Data Management Team entwickelt wurde, um den Anforderungen der vielfältigen im Projekt verwendeten Quellen gerecht zu werden. Das Datenmodell (und der anschließende Annotationsprozess) folgen dem [Handbuch zur Erstellung diskriminerungsfreier Metadaten für historische Quellen und Forschungsdaten] (https://maehr.github.io/diskriminierungsfreie-metadaten/), das von Stadt.Geschichte.Basel entwickelt wurde.

Die folgende Grafik veranschaulicht das Datenmodell mit Metadatenfeldern für ein Beispiel-Metadatenobjekt `sgb01313`, das ein untergeordnetes Medienobjekt `m01313` hat. Wenn ein Metadatenobjekt mehr als ein untergeordnetes Medienobjekt hat, werden die `id`-Nummern der untergeordneten Objekte fortlaufend nummeriert: `m01313_1`, `m01313_2` usw.
<img width="563" alt="Screenshot 2024-12-11 at 11 29 06" src="https://github.com/user-attachments/assets/2bdcddc3-3a33-45ff-aa45-b3aab9bded19">

```

**Comment by mtwente** [comment]
[Mermaid chart](https://mermaid.live/edit#pako:eNqtVEtv2zAM_iuChiAp4AROnIfjAjusAbZiHbKtww6Le1Bs2tEmS4YeaNMg_32087C7DsEOu5gU-ZH8SEre0USlQCOaCGbMgrNcsyKWKdeQWK4kufsay05nF0tCuOQ2IrVKSNduoIBuRLprZqDrta3fmeZsLcB0z3B0lZoXTG9vlFC6inuTZdkprnF_gyfbQAI2hABeo94pnYK-gBNcwgW3gUTJ9A82EIZ-C2NBW_43yAGxrwR-9p1OLA-meoakAMtSZlnTOo4uJT2Tr_1hMAyuGrPlVkBzXBm3_olzvz7Kh8aVgkk0L6uVtMKhKJVmopWBm89M22V2fVIeSG9RkVksb81Vi3nDNuWvqBb_megq0cCs0kiGy19iQO5QkGenya00mNtV8Fa5VenWgpsN_HMEDhwuDcZuy5Y_U7pgtjnjnQNp2_0ppxPA4l8cCAHEyfTEoCAfcZ5C5a3qgsncsRxI7_Z-SabBvD8iN_iu2i1pEKwiXXeEd0tCndF45D23H9y6X5088ggcnYBtHki-bFPzfGNNqzBPQBp4sdfz_YupjCnp99-iVqBWrzqW1KMFYGqe4rOvFx_T-tnGNEI1hYw5YWOKVxuhzFl1v5UJjax24FFXVqM-_iholDFh0IqZcb2fjr-SSni0ZJJGO_pEo3Dgz4IgnMynvu9PwtCjWxqNZ4OxPxxNgxA942EY7D36rBTm9Afz6Sicjf3JeO6Hs8lo6lGtXL45l8PUP2poxWn_G2GQbQM) for the data model with SGB colours:

```mermaid
classDiagram
direction LR
%%{
  init: {
    'theme': 'base',
    'themeVariables': {
      'primaryColor': '#fff',
      'primaryTextColor': '#3a1e3e',
      'primaryBorderColor': '#3a1e3e',
      'lineColor': '#3a1e3e',
      'secondaryColor': '#ffe880',
      'tertiaryColor': '#ffe880'
    }
  }
}%%

    class metadata {
        id (sgb01313)
        title
        [subject;subject]
        description
        temporal
        [isPartOf;isPartOf] (Data DOIs)
    }
    class media {
        id (m01313)
        title
        [subject;subject]
        description
        [creator] (inkl. Link zur Institution)
        [publisher] (inkl. Link zur Institution)
        date
        temporal
        type
        format
        extent
        [source] (Quelle und Link zum Katalog)
        language (ISO 639-2 Code)
        [relation] (interne Links, GitHub-Link, weitere Information)
        rights
        license
    }
    metadata "n" --> "m" media
```

**Comment by koilebeit** [comment]
resolved by #196
